specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/managedHsm.json

```diff
@@ -764,6 +910,12 @@
             "name": "PublicNetworkAccess",
             "modelAsString": true
           }
+        },
+        "purgeDateTimeUtc": {
```

Comment:
1. All dates must be in UTC anyway, so declaring "Utc" in the property name is unnecessary and not consistent with most other providers I've seen.
2. For consistency's sake with Key Vault, could this just be `scheduledPurgeDate`: https://github.com/Azure/azure-rest-api-specs/blob/7256c18a35090d545a5a3595920c2275d3e397b3/specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/keyvault.json#L1410